components/radio/themes/spectrum.css

```diff
@@ -13,15 +13,33 @@ governing permissions and limitations under the License.
 @container (--system: spectrum) {
   .spectrum-Radio {
     /* selection indicator */
-    --spectrum-radio-gray-default: var(--spectrum-gray-600);
+    /* --spectrum-radio-gray-default: var(--spectrum-gray-600);
     --spectrum-radio-gray-hover: var(--spectrum-gray-700);
     --spectrum-radio-gray-down: var(--spectrum-gray-800);
-    --spectrum-radio-gray-focus: var(--spectrum-gray-700);
+    --spectrum-radio-gray-focus: var(--spectrum-gray-700); */
 
     /* selected selection indicator */
-    --spectrum-radio-selected-gray-default: var(--spectrum-gray-700);
-    --spectrum-radio-selected-gray-hover: var(--spectrum-gray-800);
-    --spectrum-radio-selected-gray-down: var(--spectrum-gray-900);
-    --spectrum-radio-selected-gray-focus: var(--spectrum-gray-800);
+    /* --spectrum-radio-selected-gray-default: var(--spectrum-gray-700); */
+    /* --spectrum-radio-selected-gray-hover: var(--spectrum-gray-800); */
+    /* --spectrum-radio-selected-gray-down: var(--spectrum-gray-900); */
+    /* --spectrum-radio-selected-gray-focus: var(--spectrum-gray-800); */
+
+    --spectrum-radio-button-border-color-default: var(--spectrum-gray-600);
+    --spectrum-radio-button-border-color-hover: var(--spectrum-gray-700);
+    --spectrum-radio-button-border-color-down: var(--spectrum-gray-800);
+    --spectrum-radio-button-border-color-focus: var(--spectrum-gray-700);
+
+    /* "selected" aka "checked" */
+    --spectrum-radio-button-checked-border-color-default: var(--spectrum-gray-700);
+    --spectrum-radio-button-checked-border-color-hover: var(--spectrum-gray-800);
+    --spectrum-radio-button-checked-border-color-down: var(--spectrum-gray-900);
+    --spectrum-radio-button-checked-border-color-focus: var(--spectrum-gray-800);
```

Comment:
I'm defining separate custom properties for the `:checked` (aka "selected") states here because PostCSS was throwing errors when I tried to get fancy with my selectors, _plus_, I think it's probably wise (and easier) for us to make these available to consuming applications, should they need to change them.

Reply:
Changing my "selected" references in comments to say "checked" instead.